torch/_dynamo/variables/tensor.py

```diff
@@ -768,10 +769,7 @@ def wrap(i, sub_proxy):
                 with unittest.mock.patch.object(
                     tx.fake_mode, "allow_scalar_outputs", True
                 ):
-                    return SymNodeVariable.create(
-                        tx,
-                        sub_proxy.item(),
-                    )
+                    return wrap_fx_proxy(tx, sub_proxy.item())
```

Comment:
Is this a bugfix? If so, can it be triggered outside of this PR? If so, maybe split it out as a separate PR?

Reply:
yeah, can be a separate pr

Reply:
There's a lot of complexity in this PR already, I would recommend that anything that can be separate bugfixes be split out. If it's not a separate bugfix then it's fine to keep around but I will ask more questions